src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const PRICE = days * PRICE_PER_DAY;
```

Comment:
Static constants that will not change during their use are usually called capital letters.
Everything that is calculated or can change is called a `camelCase`.
```suggestion
  const price = days * PRICE_PER_DAY;
```